ui/job-view/details/DetailsPanel.jsx

```diff
@@ -145,7 +145,7 @@ class DetailsPanel extends React.Component {
         // Cancel the in-progress fetch requests.
         this.selectJobController.abort();
       }
-      // eslint-disable-next-line no-undef
+
```

Comment:
I've moved this to `globals` so I don't have to touch this file as part of the Neutrino 9 PR. The newer ESLint we get as part of Neutrino 9, now knows that `AbortController` is part of the browser API and so doesn't complain that it's undefined. (And in the Neutrino 9 PR I'm also using `--report-unused-disable-directives` with ESLint, which would cause a failure due to this disable line being unnecessary)